src/test/resources/feature/TestScenarios.feature

```diff
@@ -0,0 +1,49 @@
+@Claudia
+Feature: Add, remove and modify items from shopping cart
+
+  Scenario: Add items to cart
+
```

Comment:
Please remove empty line after Scenario all over the file.

Reply:
Done